cluster-autoscaler/utils/utils_test.go

```diff
@@ -46,6 +47,26 @@ func TestPodSpecSemanticallyEqual(t *testing.T) {
 			},
 			result: true,
 		},
+		{
+			name: "two pods with different ENV",
```

Comment:
nit: It might be worth adding a test case to `TestSanitizePodSpec` as well.

Reply:
Good point, done.

Can you re-LGTM as it got removed after the update? :)